eventActions.js

```diff
@@ -525,8 +539,10 @@ ActionSetProp.prototype.reAddEntities = function (entitiesArr) {
  * @returns {ActionSetProp}
  */
 ActionSetProp.prototype.fromJS = function (data) {
+    // FROM JS cost the error and the whole funtionality is broken.
+    // this.refsToObjectProperty.fromJS(data.refsToObjectProperty);
```

Comment:
you need to actually create new instances using "new RefToObjectProperty()".
You can look in lines 339-346 how it is done there for ActionSetElementPropChange... Here you can do the deserialization of a list of instances analogously.

Reply:
done and I do believe is working